lib/opal/source_map.rb

```diff
@@ -19,7 +19,7 @@ def map
 
         mappings = @fragments.map do |fragment|
           mapping = nil
-          source_line   = fragment.line
+          source_line   = (fragment.line && fragment.line > 0) ? fragment.line : 1
           source_column = fragment.column
           source_code   = fragment.code
```

Comment:
Instead of checking `fragment.line` here, it would be better to check `source_line` inside the `if` in the next line, so to save some method calls and make rubocop happy, e.g. `source_line = 1 if source_line.zero?`.

Reply:
done